src/index.js

```diff
@@ -1,10 +1,15 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
+import { I18nextProvider } from "react-i18next";
```

Comment:
For code consistency, let's stick with single quotes for JavaScript code and double quotes for JSX.

Reply:
Do we want to use prettier for this and on pre-push, automatically apply it to all files?

Reply:
Sure! @anteloe go for it.